install.sh

```diff
@@ -177,6 +177,24 @@ nvm_check_global_modules() {
   fi
 }
 
+nvm_bash_completion() {
+  NVM_PROFILE=$(nvm_detect_profile)
+  case "$NVM_PROFILE" in "*/.bashrc" | "*/.bash_profile" | "*/.zshrc" )
+    if [ ! -f "$NVM_DIR/bash_completion" ]; then
+      nvm_download -s "https://raw.githubusercontent.com/creationix/nvm/$(nvm_latest_version)/bash_completion" -o "$NVM_DIR/bash_completion" || {
+        echo >&2 "Failed to download \"$NVM_SOURCE_LOCAL\""
```

Comment:
There's no need to escape the double quotes here - just `echo >&2 "Failed to download "$NVM_SOURCE_LOCAL""` should work.